src/components/Discussion/Postbox/Postbox.tsx

```diff
@@ -0,0 +1,357 @@
+import { useState, useRef, useContext, useEffect, KeyboardEvent } from 'react';
+import { debounce } from 'lodash';
+import { OrbisContext } from 'contexts/Guilds/orbis';
+import {
+  PostboxWrapper,
+  PostboxInput,
+  PostboxInputWrapper,
+  PostboxMentions,
+  PostboxUserMention,
+  PostboxShare,
+  ReplyTo,
+  ReplyToDetails,
+  ReplyToCancel,
+  PostboxEditMenu,
+} from './Postbox.styled';
+import {
+  didToAddress,
+  getUsername,
+  getBadgeContent,
+  highlightMentions,
+} from 'utils/orbis';
+import { Box } from 'components/primitives/Layout';
+import { useTranslation } from 'react-i18next';
+
+const Postbox = ({
+  context,
+  replyTo = null,
+  editPost = null,
+  enterToShare = false,
+  hideShareButton = false,
+  placeholder = 'Thoughts?...',
+  callback,
+  cancelEdit,
+  cancelReplyTo,
+}: {
+  context: string;
+  replyTo?: any;
+  editPost?: any;
+  enterToShare?: boolean;
+  hideShareButton?: boolean;
+  placeholder?: string;
+  callback: (value: any) => void;
+  cancelEdit?: () => void;
+  cancelReplyTo?: () => void;
+}) => {
+  const { t } = useTranslation();
+  const { orbis, profile } = useContext(OrbisContext);
+
+  const postboxArea = useRef<any>(null);
+
+  const [focusOffset, setFocusOffset] = useState<number>(0);
+  const [focusNode, setFocusNode] = useState<Node | null>(null);
+  const [isSearching, setIsSearching] = useState(false);
+  const [searchText, setSearchText] = useState('');
+  const [searchResults, setSearchResults] = useState<any[]>([]);
+  const [mentions, setMentions] = useState<any[]>([]);
```

Comment:
Any 👀

Reply:
Updated and fixed using [types.orbis.d.ts](https://github.com/ParallaxNetwork/DAVI/blob/e2de23c5426312a0123be806072642cbbbb2d18d/src/types/types.orbis.d.ts)